src/System.Memory/dir.props

```diff
@@ -2,10 +2,10 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.1.0</AssemblyVersion>
+    <AssemblyVersion>4.0.2.0</AssemblyVersion>
     <!-- System.Memory has forwarded types into the runtime on netcoreapp/uap
          It must win over assemblies versioned at 4.0.* -->
-    <AssemblyVersion Condition="'$(TargetGroup)' == 'netcoreapp' OR '$(TargetGroup)' == 'uap' ">4.1.0.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(TargetGroup)' == 'netcoreapp' OR '$(TargetGroup)' == 'uap' ">4.1.1.0</AssemblyVersion>
```

Comment:
I guess we technically don't need to version this one since it doesn't ship in the package but 🤷

Reply:
I thought we wanted to bump both based on our other conversation, but I agree we don't need to I will revert this back to 4.1.0.0 and we will freeze it there.

Reply:
Yeah, I forgot that you were only bumping the stuff in packages.